ui/src/app/modules/boards/pages/login/login.page.html

```diff
@@ -26,18 +26,27 @@
     <div id="app-content">
         <div class="logo-container">
             <img
-                src="/assets/RetroQuest.png"
-                title="RetroQuest Logo"
-                id="logo"
-                alt="RetroQuest"
+                src="/assets/icons/icon-72x72.png"
+                title="RetroQuest Icon"
+                id="logo-pic"
+                alt="Logo"
+                class="logo-image"
             />
+
+            <div class="logo-text-container">
```

Comment:
Can you make this link to https://fordlabs.com/?

Also, can you make this a reusable component to avoid DOM duplication?

Reply:
@tsatam - Can you please clarify which aspect of it you want to be a reusable component?

Reply:
`<div class="logo-container">`, it is reused both here and on create.page.html.